bpo-40067 : Improve error messages for multiple star expressions in assignment (#19168)
python/cpython: Misc/NEWS.d/next/Core and Builtins/2020-03-25-20-34-01.bpo-40067.0bFda2.rst

> For multiple star expressions in assignment error messages improved.

Reviewer: ```suggestion
For multiple star expressions in an assignment, the error message improved.
```
This might clarify it a bit.

> Improve the error message for multiple star expressions in an assignment

Reviewer 2: ```suggestion
Improve the error message for multiple star expressions in an assignment.
```

> For multiple star expressions in an assignment, the error message improved.

Reviewer 2: ```suggestion
Improve the error message for multiple star expressions in an assignment
```
